GliaWidgets/ColorType.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+public enum ColorType {
+    case fill(color: UIColor)
+    case gradient(colors: [CGColor])
+
+    static func == (lhs: ColorType, rhs: ColorType) -> Bool {
+        switch (lhs, rhs) {
```

Comment:
I'd like to clarify, why you didn't use `default`, do you think that we may extend this enum in future?